docs/releases/release-8-3.md

```diff
@@ -0,0 +1,417 @@
+---
+sidebar_position: 1
+---
+
+# XCP-ng 8.3
+
+We are proud to present XCP-ng 8.3, the last release of the 8.x platform.
```

Comment:
"which will be the last release" ?

Reply:
I think this should be `latest` instead.

Reply:
Done

Reply:
Oops, wrong comment. Actually I really mean `last`, and Yann proposed to improve the sentence to "which will be the last release of the 8.x platform".